verifyFiles.py

```diff
@@ -228,8 +228,8 @@ def log(message):
     logging.info(message)
 
 def main():
-        parser=argparse.ArgumentParser(description='verifyFiles.py', usage='./verifyFiles.py <input_file_name>',formatter_class=argparse.RawTextHelpFormatter)
-        parser.add_argument('file_in', help= textwrap.dedent('''Input the text file containing the path to the subject folders
+        parser=argparse.ArgumentParser(description="verifyFiles.py", usage="./verifyFiles.py <input_file_name> [-c]",formatter_class=argparse.RawTextHelpFormatter)
```

Comment:
argparse will build the usage string for you based on the parameters declared below. Your usage parameter can be removed.